### PR TITLE
fix(rtl): invitees list

### LINKED ISSUE #6401: [RTL] Spacing in user select
### Steps to reproduce

Image

### Expected behavior

Good design

### Actual behaviour

Bad design 

### Calendar app version

_No response_

### CalDAV-clients used

_No response_

### Browser

_No response_

### Client operating system

_No response_

### Server operating system

_No response_

### Web server

None

### Database engine version

None

### PHP engine version

None

### Nextcloud version

_No response_

### Updated from an older installed version or fresh install

None

### List of activated apps

_No response_

### Nextcloud configuration

_No response_

### Web server error log

_No response_

### Log file

_No response_

### Browser log

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by hamza221**
fix #6401

| b | a |
|--------|--------|
| image | image |

### CODE REVIEW

File: css/app-sidebar.scss [hunk 1]
```diff
@@ -899,8 +899,14 @@
 	align-items: center;
 	width: 100%;
 
+	&--rtl{
+		text-align: right;
```
Comment (tcitworld) [inline css/app-sidebar.scss:903]: Why not `text-align: end;`?

File: css/app-sidebar.scss [hunk 2]
```diff
@@ -899,8 +899,14 @@
...
+	}
 	// Account for avatar width (because it is position: relative)
-	padding-right: 32px;
+	&--ltr{
+		padding-right: 32px;
```
Comment (tcitworld) [inline css/app-sidebar.scss:907]: Why not just `padding-block-end`?

Comment (hamza221) [reply css/app-sidebar.scss:907]: image

Comment (hamza221) [reply css/app-sidebar.scss:907]: margin-inline-start works i think

**Pull request merged**